packages/pipeline/src/parsers/token_metadata/index.ts

```diff
@@ -26,7 +26,7 @@ function parseMetamaskTrustedToken(resp: MetamaskTrustedTokenMeta, address: stri
     const trustedToken = new TokenMetadata();
 
     trustedToken.address = address;
-    trustedToken.decimals = new BigNumber(resp.decimals);
+    trustedToken.decimals = resp.decimals === undefined ? null : new BigNumber(resp.decimals);
```

Comment:
@xianny the only real difference is that we needed to check if `decimals` is `undefined` in the `parseMetamaskTrustedToken` function, not just in `parseZeroExTrustedToken` function.

Reply:
👍 cool! sorry I missed this.